internal/ingress/controller/template/configmap.go

```diff
@@ -125,8 +125,11 @@ func ReadConfig(src map[string]string) config.Configuration {
 		delete(conf, luaSharedDictsKey)
 		lsd := splitAndTrimSpace(val, ",")
 		for _, v := range lsd {
-			v = strings.ReplaceAll(v, " ", "")
-			results := strings.SplitN(v, ":", 2)
+			results := strings.SplitN(strings.ReplaceAll(v, " ", ""), ":", 2)
```

Comment:
Is there a reason to have these calls merged in one line? I'd rather keep it split for readability.

Reply:
The error message would not be accurate, e.g. `certificate_data 100` would be `certificate_data100`. I agree that a separate line would be more readable, however we would need to use a different variable since we need the original. I couldn't think of a good name and since the new value wasn't used elsewhere I did it like that.